README.md

```diff
@@ -2,24 +2,81 @@
 
 Make your own terminal aliases easily!
 
-## Install it
+[![GitHub issues](https://img.shields.io/github/issues/99xt/runn)](https://github.com/99xt/runn/issues)
+[![GitHub contributors](https://img.shields.io/github/contributors/99xt/runn)](https://github.com/99xt/runn/graphs/contributors)
+[![GitHub forks](https://img.shields.io/github/forks/99xt/runn)](https://github.com/99xt/runn/network/members)
+[![MIT license](https://img.shields.io/npm/l/chalk)](https://github.com/99xt/runn/blob/master/LICENSE)
+[![npm version](https://img.shields.io/npm/v/runnit)]()
 
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+
+## About the Project
+
+Make your own terminal aliases easily!
+
+### build with
+
+This project was built using JavaScript
+
+## Getting Started
+
+### Prerequisites
+
+You need to install node package manager to runn this!
+
+* npm
+
+```sh
+npm install npm@latest -g
 ```
+
+### Installation
+
+1. Install the NPM packages
+
+```sh
 $ npm i -g runnit
 ```
 
-## Save it
+2. Save it
 
-```
+```sh
 $ runn cu "curl https://raw.githubusercontent.com/shalithasuranga/runn/master/bin/runn.js"
 ```
 
-## Run it!
+## Usage
 
-```
+### runn it!
+
+```sh
 $ runn cu
 ```
 
+## Roadmap
+
+See the [open issues](https://github.com/99xt/runn/issues) for a list of proposed features (and known issues).
+
+## Contributing
+
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+
 ### License
 
-MIT
```

Comment:
We can remove "See `LICENSE` for more information." and add link to License with MIT instead